node-admin/src/test/java/com/yahoo/vespa/hosted/node/admin/maintenance/acl/iptables/NATCommandTest.java

```diff
@@ -20,19 +20,23 @@
     public void sampleNATCommandIPv6() throws UnknownHostException{
         InetAddress externalIP = Inet6Address.getByName("2001:db8::1");
         InetAddress internalIP = Inet6Address.getByName("2001:db8::2");
-        String iface = "eth0";
 
-        NATCommand command = new NATCommand(externalIP, internalIP, iface);
-        Assert.assertEquals("ip6tables -t nat -A POSTROUTING -o eth0 -s 2001:db8:0:0:0:0:0:2 -j SNAT --to 2001:db8:0:0:0:0:0:1; ip6tables -t nat -A PREROUTING -i eth0 -d 2001:db8:0:0:0:0:0:1 -j DNAT --to-destination 2001:db8:0:0:0:0:0:2", command.asString());
+        String insert = NATCommand.insert(externalIP, internalIP);
+        Assert.assertEquals("ip6tables -t nat -I POSTROUTING -s 2001:db8:0:0:0:0:0:2 -j SNAT --to 2001:db8:0:0:0:0:0:1 && ip6tables -t nat -I PREROUTING -d 2001:db8:0:0:0:0:0:1 -j DNAT --to-destination 2001:db8:0:0:0:0:0:2 && ip6tables -t nat -I OUTPUT -o lo -d 2001:db8:0:0:0:0:0:1 -j DNAT --to-destination 2001:db8:0:0:0:0:0:2", insert);
+
+        String drop = NATCommand.drop(externalIP, internalIP);
+        Assert.assertEquals("ip6tables -t nat -D POSTROUTING -s 2001:db8:0:0:0:0:0:2 -j SNAT --to 2001:db8:0:0:0:0:0:1; ip6tables -t nat -D PREROUTING -d 2001:db8:0:0:0:0:0:1 -j DNAT --to-destination 2001:db8:0:0:0:0:0:2; ip6tables -t nat -D OUTPUT -o lo -d 2001:db8:0:0:0:0:0:1 -j DNAT --to-destination 2001:db8:0:0:0:0:0:2", drop);
```

Comment:
Consider refering to the ip6tables program with an absolute path - you're assuming it's in PATH.

Reply:
Yeah, I'll rewrite the command to be agnostic to the iptables executable (like the other commands in that package) - for the next PR